magento/magento2
# Sort Order "Comments History" tab comments correctly when same timestamp

Sort comment history on the admin order **Comments History** tab by `entity_id` if the timestamps are the exact same second.



### Description (*)


### Related Pull Requests


### Fixed Issues (if relevant)

1. Fixes magento/magento2#34304

### Manual testing scenarios (*)

1. Create an order with two records in the `sales_order_status_history` table with the exact same timestamp.
2. View the order in the admin and select the **Comments History** tab.  The comments should be sorted correctly by `created_at` and `entity_id`

### Questions or comments


### Contribution checklist (*)
